### ISSUE TITLE
Deep learning

### ISSUE THREAD

**Issue by bhack** [IFL-CAMP/pose_regularization#1]
Do you plan to have a pytorch or tensorflow of this function? 
So that could be directly minimized as an auxilairy loss inside a network a training time.

**Comment by bhack** [comment]
I.e. For another example of TV (not on the pose) as loss see https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/image_ops_impl.py#L2163
